app/src/main/res/values/strings.xml

```diff
@@ -394,7 +394,9 @@
     <string name="create_account">Sign up</string>
     <string name="home_drawer">Home</string>
     <string name="open_beauty_drawer" translatable="false">Open Beauty Facts</string>
+    <string name="install_beauty_drawer" translatable="false">Install Open Beauty Facts</string>
     <string name="open_food_drawer" translatable="false">Open Food Facts</string>
+    <string name="install_food_drawer" translatable="false">Install Open Food Facts</string>
```

Comment:
mmh… Why make it untranslatable ?
 <string name="install_other_app_drawer" translatable="false">Install Open Food Facts</string>

and use the flavor files to override the string
app/src/off/res/values/strings.xml
app/src/obf/res/values/strings.xml
app/src/opf/res/values/strings.xml
app/src/opff/res/values/strings.xml

Reply:
I'm not sure why the author chose to set `open_beauty_drawer`  `translatable = "false"`. I decided to simply follow the precedent. 

It seems that all the flavors except for "off" have the possibility of using `install_food_drawer` & `open_food_drawer`,  therefore `install_beauty_drawer` & `open_beauty_drawer` would be the only strings to move to `app/src/off/res/values/strings.xml`. Is this right?

Reply:
I believe I've made the necessary changes. My apologies if I did not understand.

Reply:
You can not assume that all languages in the world concatenate the noun and verb that way. You need to create a dedicated string or use placeholders (I would advise for a dedicated string since placeholder are an occasion to create bugs when they are not properly translated)

OFF: "Install Open Beauty Facts"
OBF, OPF, OPFF: "Install Open Food Facts"

Reply:
I understand the problem with inaccurate grammar across localisations when simply using "Install" and concatenating it to whichever app is to be installed, although I believe this solution would still be intelligible to the native speaker because it only involves two strings.

I can see a possible solution using placeholders as you said. For example "Install %s", where "%s" could be "Open Beauty Facts" or "Open Food Facts".

Do you believe this is the way to go?

Thanks for the feedback.

Reply:
Placeholders are not ideal, but way better than concatenation 👍